acinclude/squid-util.m4

```diff
@@ -211,7 +211,7 @@ unset squid_tmp_define
 dnl aborts with an error specified as the second argument if the first argument doesn't
 dnl contain either "yes" or "no"
 AC_DEFUN([SQUID_YESNO],[
-  AS_IF([test "$1" != "yes" -a "$1" != "no"],[AC_MSG_ERROR([$2])])
+  AS_IF([test "$1" != "yes" -a "$1" != "no"],[AC_MSG_ERROR([Bad argument for $2, which does not accept a value.])])
 ])
```

Comment:
> PR title: Polish use of SQUID_YESNO macro

Please remove "use of" from the PR title. The PR changes the macro itself (in addition to changing how it is used).

FWIW, I would title this PR "Improve SQUID_YESNO error messages". The PR description and the diff make it clear where the changes are IMO.